example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -267,10 +266,6 @@ describe('The S3 Ingest Granules workflow', () => {
 
   afterAll(async () => {
     // clean up stack state added by test
```

Comment:
Assuming I'm reading this right: 

In this instance, the granule deletion is occurring as part of a test.... which *is* a duplicate, but it's relying on the test behavior/composition to remove the data, *not* a cleanup routine.

This would be fine, except we've adopted an abort behavior:
       if (beforeAllError || subTestSetupError) throw SetupError;

which means if either of the beforeAll routines fails, this granule isn't deleted (as the test doesn't run, this granule isn't deleted and things like the Collection cleanup fail, as aa virtue of an incedental failure in another test.

I think this cleanup may be a good candidate for a try/catch/ignore, absent a refactor into smaller tests (which we probably should do).